### ISSUE TITLE
 W tensorflow/core/framework/op_kernel.cc:1192] Out of range: End of sequence 	 [[Node: IteratorGetNext = IteratorGetNext[output_shapes=[[?,?,227,227,?]], output_types=[DT_FLOAT], _device="/job:localhost/replica:0/task:0/device:CPU:0"](IteratorFromStringHandle)]]

### ISSUE THREAD

**Issue by tiffany-mgx** [tnybny/Frame-level-anomalies-in-videos#6]
Hi，tnybny,
First,Thanks for your code. when I trained model 'STAE', there is a problem as flollow:
**2018-11-29 13:20:56.276236: W tensorflow/core/framework/op_kernel.cc:1192] Out of range: End of sequence
	 [[Node: IteratorGetNext = IteratorGetNext[output_shapes=[[?,?,227,227,?]], output_types=[DT_FLOAT], _device="/job:localhost/replica:0/task:0/device:CPU:0"](IteratorFromStringHandle)]]**
 do you have this problem when you are training?

**Comment by tnybny** [comment]
This isn't an error. This is an indication that the data iterator has consumed the entire test dataset once, which should happen while calculating validation loss every few training iterations. Can you confirm that the output of the run is still as expected?

**Comment by tiffany-mgx** [comment]
Thank you tnybny, you are right.This isn't an error.The code is still running.
Thanks again for your reply,I will close this issue.
